samples/model-builder/conftest.py

```diff
@@ -195,14 +195,46 @@ def mock_run_automl_image_training_job(mock_image_training_job):
 
 @pytest.fixture
 def mock_init_custom_training_job():
-    with patch.object(aiplatform.training_jobs.CustomTrainingJob, "__init__") as mock:
+    with patch.object(aiplatform.CustomTrainingJob, "__init__") as mock:
         mock.return_value = None
         yield mock
 
 
 @pytest.fixture
 def mock_run_custom_training_job():
-    with patch.object(aiplatform.training_jobs.CustomTrainingJob, "run") as mock:
+    with patch.object(aiplatform.CustomTrainingJob, "run") as mock:
+        yield mock
+
+
+@pytest.fixture
+def mock_init_custom_container_training_job():
+    with patch.object(
+        aiplatform.training_jobs.CustomContainerTrainingJob, "__init__"
+    ) as mock:
+        mock.return_value = None
+        yield mock
+
+
+@pytest.fixture
+def mock_run_custom_container_training_job():
+    with patch.object(aiplatform.CustomContainerTrainingJob, "run") as mock:
+        yield mock
+
+
+@pytest.fixture
+def mock_init_custom_package_training_job():
+    with patch.object(
+        aiplatform.training_jobs.CustomPythonPackageTrainingJob, "__init__"
+    ) as mock:
+        mock.return_value = None
+        yield mock
+
+
+@pytest.fixture
+def mock_run_custom_package_training_job():
+    with patch.object(
+        aiplatform.training_jobs.CustomPythonPackageTrainingJob, "run"
+    ) as mock:
         yield mock
```

Comment:
Please use the same pattern [as the AutoML TrainingJob fixtures](https://github.com/googleapis/python-aiplatform/blob/83b3268e7f4fdd1852759b0d96fba564facb302c/samples/model-builder/conftest.py#L140-L193). We should mock `CustomContainerTrainingJob` and `CustomPythonPackageTrainingJob` objects, like we[ did for `CustomTrainingJob`](https://github.com/googleapis/python-aiplatform/blob/83b3268e7f4fdd1852759b0d96fba564facb302c/samples/model-builder/conftest.py#L140-L143). Then patch "run" on that mock. We should not mock `__init__`.

Reply:
Fixed.